api/main.go

```diff
@@ -573,3 +556,43 @@ func wireIdentityProvider(client client.Client, restConfig *rest.Config) authori
 	certInspector := authorization.NewCertInspector(restConfig)
 	return authorization.NewCertTokenIdentityProvider(tokenReviewer, certInspector)
 }
+
+func wireLogCacheClient(cfg *config.APIConfig) (*url.URL, *http.Client, error) {
+	logCacheURL, err := url.Parse(cfg.ServerURL)
+	if err != nil {
+		return nil, nil, fmt.Errorf("could not parse server URL: %w", err)
+	}
+	logCacheClient := &http.Client{
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{
+				InsecureSkipVerify: cfg.Experimental.ExternalLogCache.TrustInsecureLogCache, // #nosec G402
+			},
+		},
+	}
+	if !cfg.Experimental.ExternalLogCache.Enabled {
+		logCacheURL, err = url.Parse(fmt.Sprintf("https://localhost:%d", cfg.InternalPort))
+		if err != nil {
+			return nil, nil, fmt.Errorf("could not parse local log cache URL: %w", err)
+		}
+
+		tlsPath := os.Getenv("TLSCONFIG")
```

Comment:
Aren't those the certificate bundle for the api server itself? An enternal log cache cert bundle could have a different cert authority